### PR TITLE
Fix: add web team theme

### PR DESCRIPTION
**Pull request by yanksyoon**
Applicable spec: N/A

### Overview

This PR adds Canonical's own theme resource-centre to the default set of themes.

### Rationale

Web team support.

### Juju Events Changes

None.

### Module Changes

None.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by merkata — DISMISSED** [review]
lgtm

File: wordpress.Dockerfile [hunk 1]
```diff
@@ -11,34 +11,35 @@ LABEL maintainer="wordpress-charmers@lists.launchpad.net"
...
-            rm -rvf "$dir";  \
-            mkdir -p "$dir";  \
-            chown "$APACHE_RUN_USER:$APACHE_RUN_GROUP" "$dir";  \
-            chmod u=rwx,g=rx,o=rx "$dir"; \
-        done && \
-        ln -sfT /dev/stdout "$APACHE_LOG_DIR/other_vhosts_access.log" && \
-        chown -R --no-dereference "$APACHE_RUN_USER:$APACHE_RUN_GROUP" "$APACHE_LOG_DIR"
+    && apt-get --purge autoremove -y \
+    && apt-get install -y apache2 \
+    bzr \
```
Comment (arturo-seijas) [inline wordpress.Dockerfile:16]: Can you indent these lines? From `bzr`to `unzip`

File: wordpress.Dockerfile [hunk 2]
```diff
@@ -74,41 +75,41 @@ RUN wp core download --version=${VERSION}
...
-        syntaxhighlighter \
-        wordpress-importer \
-        wordpress-seo \
-        wp-font-awesome \
-        wp-lightbox-2 \
-        wp-markdown \
-        wp-mastodon-share \
-        wp-polls \
-        wp-statistics ;\
+    404page \
```
Comment (arturo-seijas) [inline wordpress.Dockerfile:78]: Same here

File: wordpress.Dockerfile [hunk 3]
```diff
@@ -74,41 +75,41 @@ RUN wp core download --version=${VERSION}
...
+    wp-lightbox-2 \
+    wp-markdown \
+    wp-mastodon-share \
+    wp-polls \
+    wp-statistics ;\
     do \
-        curl -sSL "https://downloads.wordpress.org/plugin/${plugin}.latest-stable.zip" -o "${plugin}.zip"; \
-        unzip "${plugin}.zip"; \
-        rm "${plugin}.zip"; \
+    curl -sSL "https://downloads.wordpress.org/plugin/${plugin}.latest-stable.zip" -o "${plugin}.zip"; \
```
Comment (arturo-seijas) [inline wordpress.Dockerfile:110]: Same

Comment (yanksyoon) [reply wordpress.Dockerfile]: Done, thanks!

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by mthaddon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Pull request merged**